### DISCUSSION TITLE
Github Pages auto redirects to github organizations page

### DISCUSSION THREAD

**Discussion by neonklr** [community/community#87470] [Enterprise]
### Select Topic Area

Question

### Body

I have an account in enterprise setting with multiple organizations under the enterprise account. I recently made internal repository with simple index.html in the master branch and configured github pages for the repo, using master branch and root settings.

the deployment environment was then made successfully and some random link was generated for it. Up till here, everything was fine and as expected, but as soon as I clicked the link, it opened a new blank tab and tried to load the link, and then redirected automatically to the organizations page of the enterprise account (with no visible error message(s)).

Just to cross-check this behaviour, I did the same thing with both, an internal and a private visibility, with same behaviour everytime.

After further digging into redirected URL, I saw phrases in the url similar to :- 
- reason="not allowed to browse githuburl login category"
- reasoncode=CATEGORY_DENIED
(on a side note, not sure why this error was hidden in url and was not shown for the user to understand the problem)

I feel like there is configuration issue of github pages in settings of enterprise account I am using, but not sure which settings is making it such that I can deploy pages but cannot open them.

**Comment by queenofcorgis** [comment]
Hmm that is interesting, would you mind opening a ticket with our [Support team](https://support.github.com/request) please? We'll need some info from you that you may want to share privately to investigate this further.

**Reply by neonklr** [reply]
yup, I'll do that, just that I believe my enterprise is already doing this, with not much of a resolution till now from what I have heard. was asking here in the forum just in-case someone had a public knowledge of this issue.

But you are right, I guess I should just push this issue more towards support team for the best possible help to get this issue resolved.

**Reply by queenofcorgis** [reply]
@neonklr totally understand and we're so glad you came to the community first - I'm quite curious about this myself and was hoping to see other experiences with this.
